msteams-platform/concepts/authentication/nested-authentication.md

```diff
@@ -0,0 +1,212 @@
+
+## Configure nested authentication
+
+> [!NOTE]
+> Nested authentication is in developer preview and not supported by all host environments. Developers need to check the support status using the Teams JS SDK and provide a fallback experience for unsupported environments.
+
+To use nested authentication, follow these steps:
+
+1. [Register their single page application](#register-your-single-page-application)
+1. [Add trusted brokers](#add-trusted-brokers)
+1. [Enable the native bridging](#enable-native-bridging)
+1. [Initialize public client application](#initialize-public-client-application)
+1. [Call an API](#call-an-api)
+
+### Register your single-page application
+
+You need to create a Microsoft Azure App registration for your add-in on the Azure portal. The app registration must have a name, supported account type, and SPA redirect. If your add-in requires additional app registration beyond NAA and SSO, see [Register your single-page application.](/entra/identity-platform/scenario-spa-app-registration)
+
+### Add trusted brokers
+
+To enable nested authentication, your application must actively configure one or more specialized redirect URIs to signal to the Microsoft identity platform that it's allowed to be brokered by supported hosts. The redirect URI of the application must be of type **Single Page Application** and conform to the following scheme:
+
+```
+brk-<broker_application_id>://<your_domain>
+```
+
+Where <broker_application_id> is the app ID or alias of the broker or brokers you wish to trust and <your_domain> is the fully qualified domain name where your application is hosted. For example, **brk-multihub://contoso.com**.
+
+For Teams only apps, add two Redirect URIs to indicate you trust only Teams to broker tokens for you:
+
+* brk-5e3ce6c0-2b1f-4285-8d4b-75ee78787346://<your_domain>
+* brk-1fec8e78-bce4-4aaf-ab1b-5451cc387264://<your_domain>
+
+If your app has been upgraded to also run in Outlook and Microsoft365.com (in addition to Teams) then you just need to add one Redirect URI:
+
+```http
+brk-multihub://<your_domain>
+```
+
+Your domain should only include the origin and not sub-paths. For example:
+
+* Correct: brk-multihub://myapp.teams.microsoft.com
+* Incorrect: brk-multihub://myapp.teams.microsoft.com/go
+
+### Enable native bridging
+
+Configure your add-in to use nested app authentication by setting the `supportsNestedAppAuth` property to `true` in your MSAL configuration. This enables MSAL to use APIs on its native application host such as Outlook to acquire tokens for your application. If you don't set this property, MSAL uses the default JavaScript-based implementation to acquire tokens for your application, which might lead to unexpected auth prompts and unsatisfiable conditional access policies when running inside of a webview.
```

Comment:
Setting the supportsNestedAppAuth property is no longer a requirement and can be removed. MSAL has made updates to assume this based on the public client that the developer instantiates

Reply:
Removed the heading Enable native bridging.